packages/asc-ui/src/components/Header/HeaderLogoTextStyle.ts

```diff
@@ -1,6 +1,6 @@
 import styled from 'styled-components'
 
-const HeaderLogoTextStyle = styled.h1`
+const HeaderLogoTextStyle = styled.div`
```

Comment:
You'd probably want this to be configurable, because on a homepage, it makes sense to have it be an `h1`.

Reply:
It make sense but not to wrap the site logo in it. This is the wcag suggestion. Besides that this control is actualy rendered in a <header> html element that makes sense.